src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoTest.java

```diff
@@ -157,6 +161,38 @@ private void assertThatDatasetAndTablesShouldExist(Dataset dataset, boolean shou
         }
     }
 
+
+    private String insertExample = "INSERT INTO `broad-jade-dev.datarepo_hca_ebi.datarepo_load_history_staging_x` " +
```

Comment:
This string should have backslashes in it, I think.

Reply:
yes--I just cant get ST to pick them up

I think your """ idea is the best option!